src/model/Space.cpp

```diff
@@ -2404,12 +2408,20 @@ namespace model {
 
           for (Surface otherSurface : otherSurfaces) {
             std::string otherSurfaceHandle = toString(otherSurface.handle());
+            std::string otherSurfaceName = otherSurface.nameString();
+            std::string otherSurfaceType = otherSurface.surfaceType();
+
             if (hasSubSurfaceMap.find(otherSurfaceHandle) == hasSubSurfaceMap.end()) {
               hasSubSurfaceMap[otherSurfaceHandle] = !otherSurface.subSurfaces().empty();
               hasAdjacentSurfaceMap[otherSurfaceHandle] = otherSurface.adjacentSurface().has_value();
             }
 
             if (hasSubSurfaceMap[otherSurfaceHandle] || hasAdjacentSurfaceMap[otherSurfaceHandle]) {
+
+              auto subSurfaces = otherSurface.subSurfaces();
+              for (auto subSurface : subSurfaces) {
+                std::string subSurfaceName = subSurface.nameString();
+              }
```

Comment:
1. This does nothing so it's useless
2. If it did do something, this creates a copy for each iteration, use const auto& or auto&

Reply:
I removed it in db79456fe87054ed8957888e22f329598bdcec6f

Reply:
I think this was some debug code that I didn't remove